modules/andes-core/broker/src/main/java/org/wso2/andes/amqp/QpidAndesBridge.java

```diff
@@ -488,7 +469,11 @@ private static void addLocalSubscriptionsForAllBindingsOfQueue(AMQQueue queue, S
                         Andes.getInstance().openLocalSubscription(subscriptionEvent);
                         alreadyAddedSubscriptions.add(subscriptionEvent);
                     }
+                } else {
+                    throw new AndesException("Binding already exists for the queue " + queue
+                            .getName());
```

Comment:
This is QpidBridge class. Also we throw this exception due to an limitation in andes code. This is not a violation of amqp spec. So I feel like throwing an Andes exception is better here. WDYT?